switchboard/cli/Cargo.toml

```diff
@@ -27,6 +27,3 @@ chrono = "0.4.38"
 xdg = "2.5.2"
 ssh2 = "0.9.4"
 base64 = "0.22.1"
-
-[dependencies.tml-switchboard]
-path = "../switchboard"
```

Comment:
This change is not related, but we were pulling in a bunch of other dependencies into the CLI through this false dependency on the switchboard. Things work file with this removed.

Reply:
Split out into #45.